pkg/config/application.go

```diff
@@ -217,7 +250,7 @@ func (c *ConfigTx) GetAnchorPeers(orgName string) ([]Address, error) {
 	return anchorPeers, nil
 }
 
-// AddApplicationOrg adds an organization to an existing config's Application configuration.
+// AddApplicationOrg adds an organization to an existing Application configuration.
 // Will not error if organization already exists.
```

Comment:
I know this isn't part of your commit, but I'm wondering if maybe AddApplicationOrg SHOULD error if org already exists. I can't remember if this is consistent with other add/remove behavior in the rest of the package. We can always revisit this when we do some final passes over the package to clean it up

Reply:
While adding orderer org, AddOrdererOrg doesn't judge if org already exists. However, while adding endpoint or anchor peer, the corresponding function returns error if it already exist. Seems to unify this behavior for consistency.